src/d3a/setup/two_sided_pay_as_clear/test_clearing_energy.py

```diff
@@ -41,22 +39,32 @@ def get_setup(config):
             Area(
                 'House 1',
                 [
-                    Area('H1 General Load', strategy=LoadHoursStrategy(avg_power_W=100,
-                                                                       hrs_per_day=24,
-                                                                       hrs_of_day=list(
-                                                                           range(24)),
-                                                                       final_buying_rate=35),
+                    Area('H1 General Load1', strategy=LoadHoursStrategy(
```

Comment:
Sorry, I dont understand why these changes to the setup file are needed. Can you please explain? Maybe also offline!?